src/config/index.ts

```diff
@@ -18,24 +18,26 @@ export const COOKIE_DOMAIN = getEnvironmentValue("COOKIE_DOMAIN");
 export const CACHE_SERVER = getEnvironmentValue("CACHE_SERVER");
 export const SHOW_SERVICE_OFFLINE_PAGE = getEnvironmentValue("SHOW_SERVICE_OFFLINE_PAGE");
 
-// ROUTING PATH
-export const REGISTER_AN_OVERSEAS_ENTITY_URL = "/register-an-overseas-entity";
-
-export const BENEFICIAL_OWNER_TYPE_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + "/beneficial-owner-type";
-export const BENEFICIAL_OWNER_OTHER_URL = REGISTER_AN_OVERSEAS_ENTITY_URL +  "/beneficial-owner-other";
-export const ENTITY_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + "/entity";
-export const LANDING_URL = REGISTER_AN_OVERSEAS_ENTITY_URL;
-export const MANAGING_OFFICER_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + "/managing-officer";
-export const PRESENTER_URL = REGISTER_AN_OVERSEAS_ENTITY_URL + "/presenter";
-
-// TEMPLATE PATH
+// TEMPLATEs
 export const BENEFICIAL_OWNER_OTHER_PAGE = "beneficial-owner-other";
 export const BENEFICIAL_OWNER_TYPE_PAGE = "beneficial-owner-type";
 export const ENTITY_PAGE = "entity";
 export const LANDING_PAGE = "landing";
 export const MANAGING_OFFICER_PAGE = "managing-officer";
 export const PRESENTER_PAGE = "presenter";
+export const BENEFICIAL_OWNER_INDIVIDUAL_PAGE = "beneficial-owner-individual";
```

Comment:
I think we have decided to arranged these alphabetically.

Reply:
👍  will change.